src/lib/components/viewer/ViewerContext.svelte

```diff
@@ -173,7 +173,7 @@ layouts, stories, and tests.
       };
 
       task.promise.catch((error) => {
-        console.error(error);
+        console.warn(error);
```

Comment:
Rendering PDFs is on the critical path. I think we want to keep this error elevated.